vault/config.go

```diff
@@ -141,6 +141,7 @@ type ProfileSection struct {
 	WebIdentityTokenFile    string `ini:"web_identity_token_file,omitempty"`
 	WebIdentityTokenProcess string `ini:"web_identity_token_process,omitempty"`
 	STSRegionalEndpoints    string `ini:"sts_regional_endpoints,omitempty"`
+	EndpointURL             string `ini:"endpoint_url,omitempty"`
```

Comment:
Could you also please expand the test for config? 
https://github.com/ByteNess/aws-vault/blob/main/vault/config_test.go